src/main/java/com/nerdwin15/stash/webhook/Notifier.java

```diff
@@ -177,21 +186,25 @@ public NotificationResult call() throws Exception {
    * @param cloneUrl The repository url
    * @param strRef The branch ref related to the commit
    * @param strSha1 The commit's SHA1 hash code.
+   * @param targetBranch Target branch can be used in merge
    * @param omitHashCode Defines whether the commit's SHA1 hash code is omitted
    *        in notification to Jenkins.
    * @param omitBranchName Defines whether the commit's branch name is omitted
+   * @param omitTargetBranch Omit the target branch
    * @return The notification result.
    */
   public @Nullable NotificationResult notify(@Nonnull Repository repo, //CHECKSTYLE:annot
       String jenkinsBase, boolean ignoreCerts, String cloneType, String cloneUrl,
-      String strRef, String strSha1, boolean omitHashCode, boolean omitBranchName) {
+      String strRef, String strSha1, String targetBranch, boolean omitHashCode, 
+      boolean omitBranchName, boolean omitTargetBranch) {
```

Comment:
I feel the omit bools aren't clear enough. I think we should prefer names like shouldOmitTargetBranch. To me, it makes statements like `if (!shouldOmitTargetBranch)` more clear.